src/compo/icartt_nc2ioda.py

```diff
@@ -0,0 +1,259 @@
+        """
+        Make a dictionary of obsvars
+        """
+        self.obsvars = obsvars
+
+    def make_AttrData(self):
+        """
+        Make a dictionary of AttrData based on obsvars
+        """
+        AttrData = {
+            'converter': os.path.basename(__file__),
+            'nvars': np.int32(len(self.obsvars))
+        }
+        self.AttrData = AttrData
+
+    def make_varDict(self):
+        """
+        """
+        self.varDict = defaultdict(lambda: defaultdict(dict))
+        for item in self.obsvars:
+            self.varDict[item]['valKey'] = item, iconv.OvalName()
+            self.varDict[item]['errKey'] = item, iconv.OerrName()
+            self.varDict[item]['qcKey'] = item, iconv.OqcName()
+
+    def make_varAttrs(self):
+        """
+        Assign attribute to each self.obsvar
+        """
+        self.varAttrs = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
+        for item in self.obsvars:
+            self.varAttrs[item, iconv.OvalName()]['units'] = self.obsvars[item][1]
+            self.varAttrs[item, iconv.OerrName()]['units'] = self.obsvars[item][1]
+            self.varAttrs[item, iconv.OqcName()]['units'] = 'unitless'
+
+
+def get_parser():
+    """
+    Get the parser object for this script.
+    Returns:
+        parser (ArgumentParser): ArgumentParser which includes all the parser information.
+    """
+
+    # get command line arguments
+    parser = argparse.ArgumentParser(
+        description=(
+            'Reads TROPESS CO netCDF files provided by NASA ??'
+            'and converts into IODA formatted output files. Multiple'
+            'files are able to be concatenated.'),
+        formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    parser.print_usage = parser.print_help
+
+    required = parser.add_argument_group(title='required arguments')
+    required.add_argument(
+        '-i', '--input',
+        help="path of TROPESS L2 CO observation netCDF input file(s)",
+        type=str,
+        nargs='+',
+        required=True)
+    required.add_argument(
+        '-o', '--output',
+        help="path of IODA output file",
+        type=str,
+        required=True)
+
+    optional = parser.add_argument_group(title='optional arguments')
+    optional.add_argument(
+        "--levs", '--levels',
+        help="User levels. [default: %(default)s] ",
+        dest="userLevels",
+        required=False,
+        type=int,
+        nargs='+',
+        default=[0, 4])
+
+    optional.add_argument(
+        '-n', '--thin',
+        help="percentage of random thinning from 0.0 to 1.0. Zero indicates"
+        " no thinning is performed. (default: %(default)s)",
+        type=float, default=0.0)
+
+    return parser
+
+
+def main():
+
+    locationKeyList = [
+        ("latitude", "float", "degrees_north"),
+        ("longitude", "float", "degrees_east"),
+        ("pressure", "float", "Pa"),
+        ("dateTime", "long", "seconds since 1970-01-01T00:00:00Z"),
```

Comment:
```suggestion
        ("dateTime", "long", iso8601_string),
```

this string is in the `def_jedi_utils` too